IBM Cloud/WML/notebooks/unstructured_image/keras/Watson OpenScale Explanation for Image Binary Classification.ipynb

```diff
@@ -1,1058 +1 @@
rce":["target = Target(\n","    target_type=TargetTypes.SUBSCRIPTION,\n","    target_id=subscription_id\n",")\n","parameters = {\n","    \"min_feedback_data_size\": 20\n","}\n","quality_details = wos_client.monitor_instances.create(\n","    data_mart_id=data_mart_id,\n","    background_mode=False,\n","    monitor_definition_id=wos_client.monitor_definitions.MONITORS.QUALITY.ID,\n","    target=target,\n","    parameters=parameters\n",").result\n","\n","quality_monitor_id = quality_details.metadata.id"]},{"cell_type":"markdown","metadata":{"id":"6c248e3a-5f80-4298-ac38-c97531680487"},"source":["### 5.2 Store feedback record"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"04bb3961-1e8c-4c0c-8b37-a2ba5f5d7645"},"outputs":[],"source":["feedback_dataset_id = None\n","feedback_dataset = wos_client.data_sets.list(type=DataSetTypes.FEEDBACK, \n","                                                target_target_id=subscription_id, \n","                                                target_target_type=TargetTypes.SUBSCRIPTION).result\n","print(feedback_dataset)\n","feedback_dataset_id = feedback_dataset.data_sets[0].metadata.id\n","if feedback_dataset_id is None:\n","    print(\"Feedback data set not found. Please check quality monitor status.\")"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"97b6a5d8-a088-48c8-b788-4f3ec1f33054"},"outputs":[],"source":["feedback = []\n","for i in range(10):\n","    catimagePath = \"data/validation/cats/cat.\" + str(i+1025) + \".jpg\"\n","    dogimagePath = \"data/validation/dogs/dog.\" + str(i+1025) + \".jpg\"\n","    catimg = mpimg.imread(catimagePath)\n","    catimg = resize(catimg, (90, 90))\n","    dogimg = mpimg.imread(dogimagePath)\n","    dogimg = resize(dogimg, (90, 90))\n","    catelement = {\"scoring_input\":catimg.tolist(), \"target\":0}\n","    dogelement = {\"scoring_input\":dogimg.tolist(), \"target\":1}\n","    feedback.append(catelement)\n","    feedback.append(dogelement)\n","wos_client.data_sets.store_records(feedback_dataset_id, request_body=feedback, background_mode=False)"]},{"cell_type":"markdown","metadata":{"id":"7c0437bd-37cf-48ab-9832-53eecf988f64"},"source":["### 5.3 Run Quality monitor"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"fa568b5a-dcd3-4151-83c8-2ca6ff803ab9"},"outputs":[],"source":["run_details = wos_client.monitor_instances.run(monitor_instance_id=quality_monitor_id, background_mode=False).result"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"e1f74c6b-1586-441e-8345-7bd61a4c6731"},"outputs":[],"source":["wos_client.monitor_instances.show_metrics(monitor_instance_id=quality_monitor_id)"]}],"metadata":{"kernelspec":{"display_name":"Python 3.10","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.10.13"}},"nbformat":4,"nbformat_minor":4}
```

Comment:
Please update "if you are using IBM Watson Studio or `Python 3.9.x / Python 3.10.x` if using any other tool" to have only Python 3.10.x.